ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -427,10 +406,17 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     );
   },
   loadTransactions: () => dispatch(fetchTransactionsRequest()),
-  loadWallets: () => dispatch(fetchWalletsRequest())
+  loadWallets: () => dispatch(fetchWalletsRequest()),
+  validateEmail: () => undefined // TODO: add onPress of email verification
```

Comment:
it could be more appropriate to name it something like 'checkIfEmailIsValidated', having 
the validation being asynchronous with respect to the button press (it will occur within the email client)